src/main/java/com/mide/gangsaeng/board/BoardController.java

```diff
@@ -0,0 +1,24 @@
+package com.mide.gangsaeng.board;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/board")
+public class BoardController {
+
+    private final BoardService boardService;
+
+    @Autowired
```

Comment:
Can we remove this `@Autowired` annotation?

Reply:
Yes, when a class has only one constructor or Spring 4.3 version onwards, but in this project, i will write `@Autowired` for explicitness.

Reply:
Sorry, upong searching, when spring 4.3 version onwards `and` a class has only one constructor, we can remove `@Autowired` annotation.
You can check this contents in spring docs. 
[https://docs.spring.io/spring-framework/reference/core/beans/annotation-config/autowired.html](url)